contracts/Account.sol

```diff
@@ -743,6 +862,83 @@ contract Account is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Managed, I
             uint256
         )
     {
-        return (1, 1, 2, 0);
+        return (1, 2, 0, 0);
+    }
+
+    /**
+     * @notice Returns the toVote amount of CELO directed towards `group`. This is
```

Comment:
Should `toVote`, `toRevoke`, etc. be in backticks (`` ` ``) for consistency? If so, look through other comments as well.